lib/ui/pages/account-linking/account_selection_screen.dart

```diff
@@ -76,7 +76,8 @@ class AccountSelectionScreen extends StatelessWidget {
                 color: Colors.white,
                 fontSize: 20,
               ), onTap: () {
-            // TODO(kkzeng): Proceed to consent request stage
+            controller.initiateConsentRequest(
+                _accountSelectionScreen.getSelectedAccounts());
           });
```

Comment:
nit but I think there's a minor indentation issue here, maybe could add a trailing comma after the curly brace